src/test/java/com/wikia/webdriver/testcases/mercurytests/CategoryTest.java

```diff
@@ -21,14 +23,30 @@
   private void init() {
     this.category = new Category(driver);
 
-    new Navigate(driver).toPage(MercurySubpages.MAIN_PAGE);
+    new Navigate(driver).toPage(MercurySubpages.CATEGORY_TEST_PAGE);
   }
 
-  @Test(groups = "mercury_category_expandAndNavigateToCategoryPage")
-  public void mercury_category_expandAndNavigateToCategoryPage() {
+  @Test(groups = "mercury_category_expandAndNavigateToCategoryPageWithDescription")
+  public void mercury_category_expandAndNavigateToCategoryPageWithDescription() {
     init();
+    CategoryPageWithDescription categoryPage = new CategoryPageWithDescription();
 
-    category.toggle();
-    category.openCategoryPage(0);
+    category
+        .toggle()
+        .openCategoryPage(MercurySubpages.CATEGORY_WITH_DESCRIPTION);
+
+    categoryPage.check();
+  }
+
+  @Test(groups = "mercury_category_expandAndNavigateToCategoryPageWithoutDescription")
+  public void mercury_category_expandAndNavigateToCategoryPageWithoutDescription() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)